astropy/wcs/wcs.py

```diff
@@ -129,6 +130,8 @@
 # Additional relax bit flags
 WCSHDO_SIP = 0x10000
 
+# Regular expression defining SIP keyword
+SIP_KW = re.compile('''^(A|B|AP|BP)_(?:[0-9]|1[0-9])_(?:[0-9]|1[0-9])(|[A-Z])$''')
```

Comment:
@nden I meant coefficient names. One thing is, as you mentioned, trailing letter, the other: I don't see any indication that 19 is the maximum value.